compiler/src/iree/compiler/GlobalOptimization/GeneralizeLinalgNamedOps.cpp

```diff
@@ -41,17 +44,16 @@ static bool isConvFoldableToContraction(linalg::LinalgOp linalgOp) {
 
   if (!llvm::all_of(convDims.strides,
                     [](int64_t element) { return element == 1; })) {
+    LLVM_DEBUG(DBGS() << "conv not foldable: non-unit strides\n");
     return false;
   }
 
-  // Dont generalize depthwise convolutions.
-  if (!convDims.depth.empty()) {
-    return false;
-  }
-
-  // Dont generalize pooling operations. For pooling ops, the input/output
-  // channel size will be categorized as the additional batch dimension
+  // Dont generalize pooling operations or depthwise convolutions. For pooling
+  // ops, the input/output channel size will be categorized as the additional
+  // batch dimension
```

Comment:
since you touch the comment, can you help add a period at the end of the sentence?